actionlib_tools/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(actionlib_tools)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+catkin_install_python(PROGRAMS scripts/axclient.py scripts/axserver.py scripts/dynamic_action.py scripts/library.py
```

Comment:
Since rosrun calls will have to update anyway, consider using this opportunity to drop the py suffixes?

Reply:
And add a proper setup.py for the "library"...

Reply:
Yeah, `dynamic_action` and `library` (...) should be moved into a `src` directory, like proper ROS/Python packages. I was looking at refactoring/cleaning this code while I was moving things, but thought it may be out of scope for this PR (which was inspired by packaging problems).

Reply:
@ianjfrosst: I'm a fan of small PRs myself, so it would be OK for me to leave the renaming/reshuffling of the modules and script(s) out of this PR.

As long as you promise to submit a follow-up PR :)

Reply:
Yeah, I can definitely submit a follow-up for cleaning up some of the code here. I'm a little busy, but I can get it in sometime in the next couple weeks.